Standards/scs-0214-v1-k8s-node-distribution.md

```diff
@@ -80,15 +80,27 @@ If the standard is used by a provider, the following decisions are binding and v
   can also be scaled vertically first before scaling horizontally.
 - Worker node distribution MUST be indicated to the user through some kind of labeling
   in order to enable (anti)-affinity for workloads over "failure zones".
+- To provide data about this node distribution and be able to test and verify compliance with this standard,
+  providers MUST label their K8s nodes with the labels "topology.kubernetes.io/region",
+  "topology.kubernetes.io/zone" and "topology.scs.openstack.org/host-id"
+  in order to provide the expected data on the KaaS layer.
+  "topology.kubernetes.io/region" and "topology.kubernetes.io/zone" correspond with the labels
+  described in the [K8s labels documentation][k8s-labels-docs] and provide logical zones of failure
+  as well as a domain made up of one or more zones. How these are defined is up to the provider and
+  the fields are most of the time autopopulated by either the kubelet, the cloud controller or another
+  external mechanism.
+  "topology.scs.openstack.org/host-id" is an SCS-specific label, which MUST contain the hostID of the
+  physical machine running the hypervisor and not the hostID of a virtual machine.
```

Comment:
Three things:

- I would prefer more structure (use a list for the labels)
- then state for each label what the value should be (even though it's not authoritative for region and zone)
- maybe don't narrow it down to verifying compliance so much; it's also desirable in its own right for maximum transparency (and it seems t be quite common already, maybe for this very reason)

Reply:
Oh sorry, I didn't mean concrete values, just an explanation what kind of information we expect, namely the zone and the region -- you know, this may sound redundant, but I want to be extra clear that the label is what the label is :)

Reply:
Ah ok got it